streams/src/main/java/org/apache/kafka/streams/KafkaStreaming.java

```diff
@@ -29,6 +29,7 @@
 import org.slf4j.LoggerFactory;
 
 import java.util.List;
+import java.util.UUID;
```

Comment:
How does the consumer group protocol supports such debugging? Identifying a problematic host is general enough to be considered by KafkaConsumer implementation.